client/routes/workflow/helpers/event-full-transforms.spec.js

```diff
@@ -8,10 +8,10 @@ describe('eventFullTransforms', () => {
       beforeEach(() => {
         event = {
           workflowTaskCompletedEventId: 'workflowTaskCompletedEventIdValue',
-          details: [
-            'sideEffectIdValue',
-            'eyAiaGVsbG8iOiAid29ybGQiIH0', // { "hello": "world" }
```

Comment:
Check all places where you see strings like this. Most likely it is also `Payloads` now.